lib/ansible/modules/network/f5/bigip_monitor_https.py

```diff
@@ -0,0 +1,604 @@
+        this value is not provided, the default C(GET /\\r\\n) will be used.
+  receive:
+    description:
+      - The receive string for the monitor call.
+  receive_disable:
+    description:
+      - This setting works like C(receive), except that the system marks the node
+        or pool member disabled when its response matches the C(receive_disable)
+        string but not C(receive). To use this setting, you must specify both
+        C(receive_disable) and C(receive).
+  ip:
+    description:
+      - IP address part of the IP/port definition. If this parameter is not
+        provided when creating a new monitor, then the default value will be
+        '*'.
+  port:
+    description:
+      - Port address part of the IP/port definition. If this parameter is not
+        provided when creating a new monitor, then the default value will be
+        '*'. Note that if specifying an IP address, a value between 1 and 65535
+        must be specified
+  interval:
+    description:
+      - The interval specifying how frequently the monitor instance of this
+        template will run. If this parameter is not provided when creating
+        a new monitor, then the default value will be 5. This value B(must)
+        be less than the C(timeout) value.
+  timeout:
+    description:
+      - The number of seconds in which the node or service must respond to
+        the monitor request. If the target responds within the set time
+        period, it is considered up. If the target does not respond within
+        the set time period, it is considered down. You can change this
+        number to any number you want, however, it should be 3 times the
+        interval number of seconds plus 1 second. If this parameter is not
+        provided when creating a new monitor, then the default value will be 16.
+  time_until_up:
+    description:
+      - Specifies the amount of time in seconds after the first successful
+        response before a node will be marked up. A value of 0 will cause a
+        node to be marked up immediately after a valid response is received
+        from the node. If this parameter is not provided when creating
+        a new monitor, then the default value will be 0.
+  target_username:
+    description:
+      - Specifies the user name, if the monitored target requires authentication.
+  target_password:
+    description:
+      - Specifies the password, if the monitored target requires authentication.
+  partition:
+    description:
+      - Device partition to manage resources on.
+    required: False
+    default: 'Common'
+notes:
+  - Requires the f5-sdk Python package on the host. This is as easy as pip
+    install f5-sdk.
+  - Requires BIG-IP software version >= 12
+requirements:
+  - f5-sdk >= 2.2.3
+extends_documentation_fragment: f5
+author:
+  - Tim Rupp (@caphrim007)
+'''
+
+EXAMPLES = '''
+- name: Create HTTPS Monitor
+  bigip_monitor_https:
+      state: "present"
```

Comment:
Most of these examples use quoted strings where it is not required.
We tend to only quote strings that requires quoting (i.e. integers as string, colon-space combo, date formats, etc...)
If we expect people to pick up how YAML works it helps to only quote when it is required.

Reply:
@dagwieers by convention we quote all of the F5 example strings exactly because users do not know "where it is required"

Reply:
Well, then they will never really know and think they always have to quote. I don't think it's better to teach them to always quote, and certainly not using double quotes.

Besides, you're not being consistent, you're not quoting the task name value and delegate_to.

Reply:
I'm being consistent across the list of parameters to the module; not the task level params ;-)

Reply:
And what does that teach our users ?

That's why I prefer consistency across the board with a very simple rule for contributors. Only quote when it is necessary (and preferably use single quotes if you have to).

You can find some information to why in the Windows documentation (soon to be released): https://github.com/ansible/community/wiki/Windows%3A-documentation#yaml-syntax-preferred

I understand that not in all cases you have these complexities, but it becomes worse if we have to teach users different rules depending on the platform used, or the technology domain.

Reply:
Hi
Use of quotes isn't a debate for this PR, though is something that may be worth a proposal.
As we don't have this documented (`dev_guide`) there are no official rules, so this module is correct, and consistent.